Web/Scotty/Action.hs

```diff
@@ -132,6 +132,39 @@ actionErrorHandler = Handler $ \case
   AENext -> next
   AEFinish -> return ()
 
+-- | Default, unscented handler for exceptions from scotty
```

Comment:
what does unscented mean here? please clarify

Reply:
I removed the word